### PR TITLE
Add fix to resolve Ballerina keyword conflicts

### PR DESCRIPTION
**Pull request by AnjanaSenanayake**
Add fix to resolve Ballerina keyword conflicts

### CODE REVIEW

File: native/fhir-to-bal-lib/src/main/java/org/wso2/healthcare/fhir/ballerina/packagegen/tool/modelgen/ResourceContextGenerator.java [hunk 1]
```diff
@@ -78,6 +78,16 @@ private void populateResourceTemplateContexts(FHIRImplementationGuide ig) {
...
+                if (structureDefinition.getType().equalsIgnoreCase("Bundle")) {
+                    continue;
+                } else if (structureDefinition.getType().equalsIgnoreCase("OperationOutcome")) {
+                    continue;
+                } else if (structureDefinition.getType().equalsIgnoreCase("CodeSystem")) {
+                    continue;
+                } else if (structureDefinition.getType().equalsIgnoreCase("ValueSet")) {
+                    continue;
+                }
+
```
Comment (sameeragunarathne) [inline native/fhir-to-bal-lib/src/main/java/org/wso2/healthcare/fhir/ballerina/packagegen/tool/modelgen/ResourceContextGenerator.java:90]: Shall we have a final Set/List to hold these special resources and check if it contains the given string. something like

`
public final Set<String> specialResourceNames = new HashSet<>(Arrays.asList("Bundle", "OperationOutcome", "CodeSystem", "ValueSet"));
`

then here you can check if it contains the resource name if it contains

**Review by isuruh15 — COMMENTED** [review]
shall we leave a comment referring the Ballerina keywords list?
https://github.com/ballerina-platform/ballerina-lang/blob/v2201.7.0/compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/LexerTerminals.java

File: native/fhir-to-bal-lib/src/main/java/org/wso2/healthcare/fhir/ballerina/packagegen/tool/modelgen/ResourceContextGenerator.java (hunk 1 above)
Comment (AnjanaSenanayake) [reply native/fhir-to-bal-lib/src/main/java/org/wso2/healthcare/fhir/ballerina/packagegen/tool/modelgen/ResourceContextGenerator.java]: Fixed in https://github.com/ballerina-platform/fhir-tools/commit/2040a1c99dcd8534a26a554453bd39936bf0f83b

**Comment by AnjanaSenanayake** [comment]
> shall we leave a comment referring the Ballerina keywords list? https://github.com/ballerina-platform/ballerina-lang/blob/v2201.7.0/compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/LexerTerminals.java

Fixed in https://github.com/ballerina-platform/fhir-tools/commit/2040a1c99dcd8534a26a554453bd39936bf0f83b

**Review by isuruh15 — APPROVED** [review]

**Pull request merged**